editor/animation_track_editor.h

```diff
@@ -34,6 +34,7 @@
 #include "editor/editor_data.h"
 #include "editor/editor_properties.h"
 #include "editor/property_selector.h"
+#include "scene/3d/mesh_instance_3d.h"
```

Comment:
This should be in cpp file.
Or you can use `is_class()` to avoid including it at all.